src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue8964.cs

```diff
@@ -1,4 +1,5 @@
-﻿#if TEST_FAILS_ON_CATALYST && TEST_FAILS_ON_WINDOWS && TEST_FAILS_ON_ANDROID //The Position property now functions correctly on Android and Windows, issue: https://github.com/dotnet/maui/issues/15443. Note that on Catalyst, swipe and drag options are not supported in Appium.
+﻿#if TEST_FAILS_ON_CATALYST && TEST_FAILS_ON_WINDOWS && TEST_FAILS_ON_ANDROID && TEST_FAILS_ON_IOS //The Position property now functions correctly on Android and Windows, issue: https://github.com/dotnet/maui/issues/15443. Note that on Catalyst, swipe and drag options are not supported in Appium.
```

Comment:
Now, on Catalyst, the Appium drag actions should be supported. However, probably you will get the same result than on iOS. But, to avoid future confusion, could update the comment?

Reply:
I have modified the comments; kindly review the changes and let me know if you have any concerns.